techniques/general/G88.html

```diff
@@ -54,46 +52,43 @@
 
 <div class="default-grid with-gap leftcol">
 <aside class="box nav-hack sidebar standalone-resource__sidebar">
-    <header class="box-h">Page Contents</header>
+    <header class="box-h">このページの内容</header>
     <div class="box-i">
       <nav aria-label="page contents" class="navtoc">
-        <ul><li><a href="#technique">About this Technique</a></li>
-<li><a href="#description">Description</a></li>
-<li><a href="#examples">Examples</a></li>
-<li><a href="#related">Related Techniques</a></li>
-<li><a href="#tests">Tests</a></li>
-<li><a href="#test-rules">Test Rules</a></li>
+        <ul><li><a href="#technique">このテクニックについて</a></li>
+<li><a href="#description">解説</a></li>
+<li><a href="#examples">事例</a></li>
+<li><a href="#related">関連テクニック</a></li>
+<li><a href="#tests">検証</a></li>
+<li><a href="#test-rules">テストルール</a></li>
 </ul>
       </nav>
     </div>
 </aside>
 
 <main id="main" class="standalone-resource__main">
-<!-- 日本語訳されるまでの一時的な注記 -->
-<div class="untranslated-note"><p>このWCAG 2.2 テクニック集の日本語訳は作業中となっています。WCAG 2.1 達成方法集の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Techniques/">WCAG 2.1 達成方法集</a> ]</p></div>
 		
-<h1><span>Technique G88:</span>Providing descriptive titles for web pages</h1>
+<h1><span>テクニック G88:</span>ウェブページに説明的なタイトルを提供する</h1>
 
 
 
 
 <section id="technique" class="box">
-	<h2 class="box-h box-h-icon">About this Technique</h2>
+	<h2 class="box-h box-h-icon">このテクニックについて</h2>
 	<div class="box-i">
 
 
   
   <p>
-    This technique relates to
+    このテクニックは
     
-<a href="https://w3c.github.io/wcag/understanding/page-titled">2.4.2: Page Titled</a>
-(<strong>Sufficient</strong>
-  using a more specific technique).</p>
+<a href="https://waic.jp/translations/WCAG22/Understanding/page-titled">2.4.2: ページタイトル</a>
+(より具体的なテクニックを用いれば<strong>十分なテクニック</strong>) に関連する。</p>
```

Comment:
Sufficient using a more specific technique の訳ですが、G131 https://github.com/waic/wcag22/pull/1197 と合わせて「より詳細なテクニックを用いれば十分なテクニック」とするのはいかがでしょうか。
```suggestion
(より詳細なテクニックを用いれば<strong>十分なテクニック</strong>) に関連する。</p>
```